MerlinAU.sh

```diff
@@ -5872,12 +5872,10 @@ _ShowMainMenu_()
    # Check for new script updates #
    if [ "$scriptUpdateNotify" != "0" ]
    then
-      forceScriptUpdateOption=""
       printf "\n ${GRNct}up${NOct}.  Update $SCRIPT_NAME Script Now"
```

Comment:
And for this one too, so they both match (i.e. no sense of immediacy).

Reply:
Fair point!

Reply:
They now both match: https://github.com/ExtremeFiretop/MerlinAutoUpdate-Router/pull/262/commits/c732d270df3fdaa1dc238ed0d7536a0cf201cb3b